src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -105,13 +121,36 @@ protected void end() {
         };
     }
 
-    public Command DriveStraight() {
-        return new SubsystemCommand(this) {
+    public Command joystickArcadeTwoStick() {
+        return new SubsystemCommand("joystick Arcade with two sticks", this) {
+            @Override
+            protected void initialize() {
+            }
+
+            @Override
+            protected void execute() {
+                m_drive.arcadeDrive(-Robot.m_oi.leftDriveStick.getY(), Robot.m_oi.rightDriveStick.getX());
+
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return false;
+            }
+
+            @Override
+            protected void end() {
```

Comment:
this applies to all your commands: don't actually need things like "end" if you aren't using them in your command. I think all you need is initialize and isFinished (not sure though, I would double check)

Reply:
Only isFinished is required. So you shouldn't include any other functions if they're empty.